src/java.base/share/classes/java/io/FilterReader.java

```diff
@@ -76,9 +76,11 @@ public int read(char cbuf[], int off, int len) throws IOException {
     }
 
     /**
-     * Skips characters.
+     * {@inheritDoc}
      *
-     * @throws     IOException  If an I/O error occurs
+     * @throws     IllegalArgumentException  If {@code n} is negative and the
```

Comment:
Does this have to be different from the `Reader.skip()`'s description? Since the contained reader implements `Reader` (throws IAE as a contract), that condition after `and` is always true?

Reply:
This came from some Readers, e.g., `CharArrayReader`, `StringReader`, overriding `skip()` _not_ to throw an IAE. But at the specification level perhaps this should not be recognized.

Reply:
Makes sense. Then I would let CSR decide whether to include the description or not.